### PR TITLE
docs(cn): fix typo

### PR DESCRIPTION
**Pull request by yionr**

### CODE REVIEW

**Comment by yionr** [comment]
@weijianfan There are instances of mixed Chinese and English terms in the article. I have standardized them to Chinese. Additionally, I found a typo and made the correction. Thank you for reviewing it.

**Comment by zloirock** [comment]
Thanks. Waiting for review from someone Chinese speaking -)

File: docs/zh_CN/2023-02-14-so-whats-next.md
Comment (cotes2020) [inline docs/zh_CN/2023-02-14-so-whats-next.md:585]:
```suggestion
我正考虑在转译器或捆绑器插件中解决这个问题，与前一个类似（但当然，更复杂一点）——我们可以将纯版本的导入替换为全局版本导入，并删除目标引擎中不必要的 polyfill。
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]

File: docs/zh_CN/2023-02-14-so-whats-next.md
Comment (cotes2020) [inline docs/zh_CN/2023-02-14-so-whats-next.md:111]: When Chinese and English words are in the same sentence, there should be a space between the Chinese and English words.

```suggestion
旧 JavaScript 引擎仍然流行，用户并不着急，还没有什么机会放弃它们，即使新的 ECMAScript 标准有着快速和可靠的优点，想让 JavaScript 引擎支持这个新标准，也要等很多年。但可以使用一些工具尝试使用新标准，让转译器（这个词不像现在这么流行）和标准库来解决语法和 polyfill 的问题。当时，这些工具包才刚刚出现。
```

File: docs/zh_CN/2023-02-14-so-whats-next.md
Comment (cotes2020) [inline docs/zh_CN/2023-02-14-so-whats-next.md:113]:
```suggestion
那时，ECMAScript 转译器开始流行并发展迅速。而与此同时，polyfill 几乎没有根据用户和实际项目的需求而发展。它们不是模块化的，并可能带来全局命名空间污染——它们不适合做库。它们不是单一复合体——而是需要用多个来自不同作者的不同的 polyfill 库，并以某种方式使它们工作在一起——在某些情况下，这几乎是不可能的。太多必要的基本语言功能都还没有实现。
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Comment by cotes2020** [comment]
Hi @zloirock, This PR adds several Chinese translations. But there is no format for mixing Chinese and English words.

Unfortunately, as of version 2af4b42 of this file, there are still many similar formatting issues and Markdown syntax errors. I can submit a new PR to fix these issues if you want.

**Comment by weijianfan** [comment]
I agree all these changes.

File: docs/zh_CN/2023-02-14-so-whats-next.md
Comment (cotes2020) [reply docs/zh_CN/2023-02-14-so-whats-next.md:585]: When Chinese and English words are in the same sentence, there should be a space between the Chinese and English words.

**Comment by yionr** [comment]
> Hi @zloirock, This PR adds several Chinese translations. But there is no format for mixing Chinese and English words.
> 
> Unfortunately, as of version [929010a](https://github.com/zloirock/core-js/commit/929010a8d4a24b176a862dbaeb60258a4014aa16) of this file, there are still many similar formatting issues and Markdown syntax errors. I can submit a new PR to fix these issues if you want.

I'm agree with you.since i have did such work a lot in another repo: zh-hans.react.dev ,there is a [guide](https://github.com/reactjs/zh-hans.react.dev/wiki/React-Translation-Guide) for typesetting,I have my first time found that it is very comprehensive and also well accepted by contributors.

But maybe it's not a common standerd? other repo related to chinese translation may have another rule.In face,in my opinion , such a "not so good" typesetting doesn't make reading more difficult for A LOT --- i mean just unconfortable for reading(as a native chinese).Objectively speaking, it does make reading easier!On another hand,if these typesetting haven't be ovserveded from beginning of translation, it will later cause a lot of time for currecting(maybe there are a tool just do this work?i don't know).Compare with the effect of translate itself,it doesn't worth for.
finnally,i found a browser extension: [为什么你们就是不能加个空格呢？](https://chrome.google.com/webstore/detail/%E7%82%BA%E4%BB%80%E9%BA%BC%E4%BD%A0%E5%80%91%E5%B0%B1%E6%98%AF%E4%B8%8D%E8%83%BD%E5%8A%A0%E5%80%8B%E7%A9%BA%E6%A0%BC%E5%91%A2%EF%BC%9F/paphcfdffjnbcgkokihcdjliihicmbpd).it's funny,it also well resolved this kind of issue(hard to unify for vary chinese translation docs' typesetting)

If the above offend you, this is not my intention, maybe my English is poor.I attach the Chinese expression to better express my meaning:

首先，我非常同意你对于本译文存在排版问题的观点。我之前在 react 文档的中文翻译仓库中也做了很多这种类型的 review，那个仓库里介绍了中文翻译应该遵守的一些规则，我是有史以来第一次觉得原来这种事情也可以描述的这么清楚还不留死角，我自那之后也很推崇这种排版方式。

但是我无法确认的是，这是一个权威的中文翻译标准吗？或许其他仓库有一个更好的标准只是我不知道？从我个人的综合理解而言，像本文目前并没有考虑到翻译排版问题的翻译结果，虽然确实会让阅读不那么舒服，但是其实也没太大问题哈哈，虽然如果事先能考虑到的话当然是更好的！

但是从后期维护的工作量而言，如果没有一个相关的工具做这件事，而只是纯人工去矫正，改起来将是十分费力的，但是从最终效果而言也没有提升多少，所以我认为这种精力的花费就很不值得了（我是在为你考虑）。

后来，我发现了一个理想的解决方案（链接在上面有），那个浏览器扩展能自动将这类排版问题矫正，由于是扩展的形式，在所有网站所有的文档都适用，你想想，这是不是形成了降维打击？我安利给你，也期待未来这类问题会有更容易的解决办法。

最后再表示一下，希望没有冒犯到你！如果你仍然觉得想做这个事情，我想也没有人会拒绝这个提议。

**Comment by cotes2020** [comment]
> ... On another hand,if these typesetting haven't be ovserveded from beginning of translation, it will later cause a lot of time for currecting(maybe there are a tool just do this type of work?i don't know).Compare with the effect of translate itself,i think it doesn't worth for.

Hi @yionr, don't worry about it, this time-consuming formatting work can and should be left to the tools.

**Comment by zloirock** [comment]
The proposed changes look logical.

**Review by zloirock — APPROVED** [review]

**Comment by zloirock** [comment]
@cotes2020
> Hi @zloirock, This PR adds several Chinese translations. But there is no format for mixing Chinese and English words.
> 
> Unfortunately, as of version [929010a](https://github.com/zloirock/core-js/commit/929010a8d4a24b176a862dbaeb60258a4014aa16) of this file, there are still many similar formatting issues and Markdown syntax errors. I can submit a new PR to fix these issues if you want.

Sure, such fixes would be very useful.

**Pull request merged**
